src/main/java/uk/gov/hmcts/ccd/auditlog/AuditEntry.java

```diff
@@ -0,0 +1,169 @@
+    private String path;
+    private String idamId;
+    private String caseId;
+    private String caseType;
+    private String jurisdiction;
+    private String eventSelected;
+    private String invokingService;
+    private String operationType;
+    private String requestId;
+    private String targetIdamId;
+    private List<String> targetCaseRoles;
+
+    public void setDateTime(String time) {
+        this.dateTime = time;
+    }
+
+    public String getDateTime() {
+        return dateTime;
+    }
+
+    public int getHttpStatus() {
+        return httpStatus;
+    }
+
+    public void setHttpStatus(int httpStatus) {
+        this.httpStatus = httpStatus;
+    }
+
+    public String getHttpMethod() {
+        return httpMethod;
+    }
+
+    public void setHttpMethod(String httpMethod) {
+        this.httpMethod = httpMethod;
+    }
+
+    public String getPath() {
+        return path;
+    }
+
+    public void setPath(String path) {
+        this.path = path;
+    }
+
+    public String getOperationType() {
+        return operationType;
+    }
+
+    public void setOperationType(String operationType) {
+        this.operationType = operationType;
+    }
+
+    public String getIdamId() {
+        return idamId;
+    }
+
+    public void setIdamId(String idamId) {
+        this.idamId = idamId;
+    }
+
+    public String getInvokingService() {
+        return invokingService;
+    }
+
+    public void setInvokingService(String invokingService) {
+        this.invokingService = invokingService;
+    }
+
+    public static String getTAG() {
+        return TAG;
+    }
+
+    public String getCaseId() {
+        return caseId;
+    }
+
+    public void setCaseId(String caseId) {
+        this.caseId = caseId;
+    }
+
+    public String getJurisdiction() {
+        return jurisdiction;
+    }
+
+    public void setJurisdiction(String jurisdiction) {
+        this.jurisdiction = jurisdiction;
+    }
+
+    public String getCaseType() {
+        return caseType;
+    }
+
+    public void setCaseType(String caseType) {
+        this.caseType = caseType;
+    }
+
+    public String getEventSelected() {
+        return eventSelected;
+    }
+
+    public void setEventSelected(String eventSelected) {
+        this.eventSelected = eventSelected;
+    }
+
+    public String getRequestId() {
+        return requestId;
+    }
+
+    public void setRequestId(String requestId) {
+        this.requestId = requestId;
+    }
+
+    public String getTargetIdamId() {
+        return targetIdamId;
+    }
+
+    public void setTargetIdamId(String targetIdamId) {
+        this.targetIdamId = targetIdamId;
+    }
+
+    public List<String> getTargetCaseRoles() {
+        return targetCaseRoles;
+    }
+
+    public void setTargetCaseRoles(List<String> targetCaseRoles) {
+        this.targetCaseRoles = targetCaseRoles;
+    }
+
+    @Override
+    public String toString() {
+        return TAG + " " + dateTime + " operationType:" + operationType +
```

Comment:
I'm thinking that maybe for consistency we should have a field id for the datetime, like `datetime:` 
This will also simplify the processing of the records.

Reply:
need to reflect in the jira/confluence first.